build.gradle

```diff
@@ -14,7 +14,7 @@ dependencies {
     implementation 'net.sf.saxon:Saxon-HE:9.8.0-14'
     implementation 'xml-resolver:xml-resolver:1.2'
 
-    testCompile('org.spockframework:spock-core:1.3-groovy-2.5') {
+    testImplementation('org.spockframework:spock-core:1.3-groovy-2.5') {
```

Comment:
https://github.com/eerohele/saxon-gradle/pull/32/commits/0cbe4379b27af1aebc35cc8e93965a07f1167d25 proposes `testCompileOnly` instead of `testImplementation`. Do you know which is correct?

Reply:
`testCompileOnly` is something that will not be on the test runtime classpath,
I'm not sure how the spok framework works but my guess is that the tests will complain about missing classes if the dependency is not `testImplementation`. This is also suggested in the spokframework examples https://github.com/spockframework/spock-example/blob/master/build.gradle#L19-L21

Reply:
I think dgeissl is correct.
setting it as testCompileOnly was incorrect.